### ISSUE TITLE
Is there a way to adjust the font range of the cloud based on screen size?

### ISSUE THREAD

**Issue by pmi123** [niconoe/django-nuages-tag#4]
In order to make a more responsive web page, I was wondering if there is a way to change the min and max font values based on the screen size? The issue I am running into is the tag cloud looks great on a large display, but on a phone's display, the fonts need to be scaled down to fit on the phone's screen. Thanks for any suggestions on how to make the tag cloud more responsive.

**Comment by pmi123** [comment]
Turns out it was not so difficult after all to make the fonts responsive in the tag cloud using the font unit **vw**. Following the example in the README.rst:

1. Set the min/max font sizes to 1 and 5, and use a linear scale. 
      `{% compute_tag_cloud my_favourite_tools interest font_size 1 5 lin %}`

2. When rendering the tag cloud, use **vw** instead of **px** for the font units
       `<span style="font-size: {{ tool.font_size }}vw;"> {{ tool.name }} </span>`

And the tag cloud fonts will change size across different devices, while maintaining the relative size difference between the tags. I found these sizes to work best, but ymmv. 1 vw=1% of the viewport size, so the fonts get really big after 5vw. Fractional vw's also work appropriately.
